docs/source/getting-started/installation.rst

```diff
@@ -46,6 +46,20 @@ If you're interested in using our visualizations, you can optionally install the
 
 We use Vue.js as our frontend web framework, and ``npm`` as our package manager.
 
+Visualization API calls
+----------------------
+On Ubuntu and other Linux flavors: if you want to use the new Augur API Calls that generate downloadable graphics developed in the `https://github.com/chaoss/augur-community-reports` repository, you need to install the Chromium ChromeDriver at the system level. This dependency exists because the Bokeh libraries we use for these APIs require a web browser engine. 
```

Comment:
`https://github.com/chaoss/augur-community-reports` is still not in a hyperlink target. Please reformat it so that it appears as a clickable link, like [this](https://github.com/chaoss/augur-community-reports), instead of just a raw URL.